src/stories/components/card-navegation/card-somethig-wrong.tsx

```diff
@@ -68,7 +67,7 @@ const TypographyDescription = styled(Typography)<{ marginBottom?: string, isLigh
   fontStyle: 'normal',
   fontWeight: 500,
   fontSize: '15px',
-  lineHeight: '18px',
+  // lineHeight: '18px',
```

Comment:
If is not necessary  , can you remove the comment here

Reply:
Removed